docs/sections/workflows/workchain.md

```diff
@@ -454,6 +458,8 @@ Maybe also use the command to show more details for the `addition()` calculation
 
 :::
 
+(workflows-workchain-context)=
+
 ## Multiple work chain steps - Context
```

Comment:
```suggestion
## Multiple work-chain steps - Context
```
Work-chain there is a compound adjective, I guess.

Reply:
Hmm, fair point! I think that's correct, but it does look a bit funny to me. :P